.gitignore

```diff
@@ -1,3 +1,3 @@
 # .gitignore
 node_modules/
-
+test.md
```

Comment:
- [ ] [OPTIOANL] Great job adding `test.md` text to the `gitignore` file but you still need to add `test.md` file to the project with the text `This file should be ignored by git` per the project requirements. Sorry it wasn't mentioned earlier by the previous reviewer. 😄